setup.py

```diff
@@ -28,6 +45,38 @@ def proto_compile(output_path):
             file.truncate()
 
 
+def install_libp2p_daemon():
+    # check go version:
+    try:
+        proc = subprocess.Popen(['go', 'version'],
+                                stdout=subprocess.PIPE)
```

Comment:
TODO find a popular/frequently used way to package and install Go dependencies for Python projects